core-api-library/library/src/main/java/net/gini/android/core/api/Resource.kt

```diff
@@ -0,0 +1,10 @@
+package net.gini.android.core.api
+
+sealed class Resource<T>(
```

Comment:
Please add the response status code and reponse headers multimap to the resource.